html.go

```diff
@@ -124,7 +124,7 @@ func (w *HTMLWriter) Newline() {
 
 // NewHTMLRenderer creates and configures an HTMLRenderer object, which
 // satisfies the Renderer interface.
-func NewHTMLRenderer(params HTMLRendererParameters) Renderer {
+func NewHTMLRenderer(params HTMLRendererParameters) *HTMLRenderer {
```

Comment:
What shall we decide for the Latex renderer? In its current state it is obviously useless. Maintaining 2+ renderers is good practice to make sure the parser interface is not too biased toward one implementation of a renderer.
Thus I would suggest someone maintains a LaTeX renderer that is on par with the HTML renderer, either in this repository or in another.

If you want to drop it from this repo, I can implement my own and reference it here.